components/http_request.rst

```diff
@@ -14,6 +14,7 @@ The ``http_request`` component lets you make HTTP/HTTPS requests. First, you nee
     # Example configuration entry
     http_request:
       useragent: esphome/device
+      id: http_request_data
```

Comment:
I don't like setting the `id` like this on the main example, especially when it is only used way down below in the new example...

Reply:
Removed